main/src/cgeo/geocaching/network/HtmlImage.java

```diff
@@ -232,7 +232,9 @@ public void run() {
                         final ImmutablePair<BitmapDrawable, Boolean> loaded = loadFromDisk();
                         final BitmapDrawable bitmap = loaded.left;
                         if (loaded.right) {
-                            emitter.onNext(bitmap);
+                            if (bitmap != null) {
```

Comment:
There is a case where the pair can be `(null, true)` in `HtmlImage#loadCachedImage()` if the image is present and fresh and we do not want to display it (`onlySave` is `true`).

In order not to mask a problem, I would rather `onNext` the bitmap only if `onlySave` is `false`, and use:

``` java
   if (!onlySave) {
      emitter.onNext(bitmap);
   }
```

so if we have a `null` image popping up while `onlySave` is `false` we get this error again (this should not happen).